src/chef-mover/apps/chef_db/src/chef_sql.erl

```diff
@@ -31,6 +31,7 @@
 
 -ifdef(TEST).
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
Does removing the export from here, rather the ifdef(TEST) macro break the tests?